app/views/stores/new.html.erb

```diff
@@ -4,21 +4,19 @@
       <h1>Store register</h1>
     </section>
     <section class="form-container">
-      <form action="">
+      <%= form_with model: @store do |form| %>
         <div class="info-container">
-          <label for="">Name</label>
-          <input type="text" name="" id="" placeholder="  Email...">
+          <%= form.label :name %><br>
+          <%= form.text_field :name %>
         </div>
         <div class="info-container">
-          <label for="">Adress</label>
-          <input type="text" name="" id="" placeholder="  Name...">
+          <%= form.label :adress %><br>
+          <%= form.text_field :adress %>
```

Comment:
Just a gramatical observation: Address is with double d 😜.

You can just update the label, since touching variables and backend could break this feature.

But LGTM.

Reply:
Ups